index.html

```diff
@@ -959,19 +959,27 @@ <h2><dfn>Coalesced events</dfn></h2>
                 <figcaption>Example of a curve in a drawing application – using only the coalesced
                 coordinates from <code>pointermove</code> events (the grey dots), the curve is noticeably
                 angular and jagged; the same line drawn using the more granular points provided by
-                <code>getCoalescedEvents</code> (the red circles) results in a smoother approximation
+                <code>getCoalescedEvents()</code> (the red circles) results in a smoother approximation
                 of the pointer movement.</figcaption>
             </figure>
 
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
-            zero or more <code>PointerEvent</code>s). If a trusted event is a <code>pointermove</code> or
-            <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
+            zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.
             Untrusted events have their <a>coalesced events list</a> initialized to the value passed to the
             constructor.</p>
 
-            <p>The events in the coalesced events list of a trusted event will have increasing
-            {{Event/timeStamp}}s, so the first event will have the smallest {{Event/timeStamp}}.</p>
+            <p>The events in the coalesced events list of a trusted event will have:</p>
+            <ul>
+                <li>Monotonically increasing {{Event/timeStamp}} values [[DOM]] - all coalesced events have a
```

Comment:
Nit (also in Sec10.2): emdash instead of dash?

Reply:
done (and checked throughout document that we use it consistently ... had a few minus signs, and one or two en-dashes)